modules/nested-tabs-html/assets/scss/frontend.scss

```diff
@@ -93,16 +137,28 @@
 		}
 
 		&-content {
+			display: var( --n-tabs-content-display );
 			flex-grow: 1;
-			padding: var( --n-tabs-content-padding );
-			border-radius: var( --n-tabs-content-border-radius );
 			min-width: 0;
+
+			> .e-con {
+
+				&:not( .e-active ) {
+					display: none;
+				}
+			}
+		}
+
+		&:not( .e-activated ) > .e-n-tabs-content > .e-con:nth-child( 1 ) {
+			display: flex;
 		}
 	}
 
 	.e-n-tab {
 
 		&-title {
+			all: unset;
```

Comment:
Remove default button styling, as done by Rami in other widgets as well.

We use `button` elements for semantic reasons, but not for styling purposes.